doc/specs/#182 - Support for installation of portable standalone apps.md

```diff
@@ -0,0 +1,190 @@
+Found NuGet [Microsoft.NuGet] Version 7.0
+This application is licensed to you by its owner.
+Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
+Downloading https://dist.nuget.org/win-x86-commandline/v7.0.0/nuget.exe
+  ██████████████████████████████   0.1 MB /  0.1 MB
+Successfully verified installer hash
+Starting package install...
+Successfully installed
+```
+
+If the portable application is running when the user performs an upgrade, the user will be informed and given options.
+
+```text
+winget upgrade Microsoft.NuGet
+Found NuGet [Microsoft.NuGet] Version 7.0
+This application is licensed to you by its owner.
+Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
+Downloading https://dist.nuget.org/win-x86-commandline/v7.0.0/nuget.exe
+  ██████████████████████████████   0.1 MB /  0.1 MB
+Successfully verified installer hash
+Starting package install...
+Failed to install NuGet [Microsoft.Nuget].
+Package in use. Either exit the program or use "--force" to upgrade.
+```
+
+### Uninstalling a portable package
+
+```text
+winget uninstall NuGet 
+Found NuGet [Microsoft.NuGet]
+Starting package uninstall...
+Successfully uninstalled
+```
+
+If the portable application created files in the portable application's directory, the user will be informed.
+
+```text
+winget uninstall NuGet 
+Found NuGet [Microsoft.NuGet]
+Starting package uninstall...
+The "--purge" argument was not specified. 
+Files still exist in "%LOCALAPPDATA%/WinGet_Packages/User/Microsoft.NuGet/". 
+Successfully uninstalled
+```
+
+## Capabilities
+
+This will allow users to manage portable applications via the Windows Package Manager.
+
+### Accessibility
+
+Any description or output text that is added by consequence of this feature will need to be localized. Moreover, this feature allows a mechanism to add accessibility settings in the future.
+
+### Security
+
+This feature will not add any new security risks. All risks associated with this feature have been resolved with the core install, import, upgrade, and uninstall features.
+
+>Note: We recommend users leverage the Windows Sandbox when testing manifests on their local machines as the programs may not have been validated from a security perspective prior to being submitted to the Windows Package Manager Community App Repository.
+
+### Reliability
+
+This should not introduce any new reliability concerns.
+
+### Compatibility
+
+Previous versions of the Windows Package Manager will not cease to function, but they will not be able to apply arguments or settings related to this feature. They will not be able to install portable applications.
+
+An update to the Windows Package Manager is required to use this feature with the related arguments and settings.
+
+### Performance, Power, and Efficiency
+
+This should not introduce any new performance, power, or efficiency concerns in the Windows Package Manager.
+
+## Potential Issues
```

Comment:
I would suggest that this issue be mentioned, as any path output in the case of remaining files may run into similar feedback -
* https://github.com/microsoft/winget-cli/issues/1977

Reply:
added